src/core/functions.ts

```diff
@@ -11,31 +9,37 @@ import type {
     AutocompleteInteraction
 } from 'discord.js';
 import { ApplicationCommandOptionType, InteractionType } from 'discord.js';
-import { PayloadType, PluginType } from './structures';
+import { PluginType } from './structures/enums';
 import assert from 'assert';
 import type { Payload } from '../types/utility';
 
-//function wrappers for empty ok / err
-export const ok = /* @__PURE__*/ () => Ok.EMPTY;
-export const err = /* @__PURE__*/ () => Err.EMPTY;
+/**
+ * Removes the first character(s) _[depending on prefix length]_ of the message
+ * @param msg
+ * @param prefix The prefix to remove
+ * @returns The message without the prefix
+ * @example
+ * message.content = '!ping';
+ * console.log(fmt(message, '!'));
```

Comment:
you're passing message as Object but portraying it should be a string?
 ```suggestion
  * console.log(fmt(message.content, '!'));
```